dm/config/subtask.go

```diff
@@ -137,6 +137,10 @@ type SubTaskConfig struct {
 	// deprecated
 	OnlineDDLScheme string `toml:"online-ddl-scheme" json:"online-ddl-scheme"`
 
+	// pt/gh-ost name rule,support regex
+	ShadowTableRule string `yaml:"shadow-table-rule" toml:"shadow-table-rule" json:"shadow-table-rule"`
```

Comment:
maybe not export `ShadowTableRule` and `TrashTableRule`  to user is bettrer for now 

we can tell if these temporary tables are from `pt` or `ghc`>